### PR TITLE
Whitelist dialog updates 

### LINKED ISSUE #3281: Implement success state in ManageWhitelistedModal
## Specification
* no specs for it yet - please talk with Karol

### Description 

* Fix error which is displaying while doing upload
* For removal we probably just need to refresh list to show proper addresses

### TODO 
- [x] Get rid of incorrect error while uploading csv file
- [x] Refresh list of whitelisted addresses when only some of them was removed (currently nothing is happening)

### PR DESCRIPTION
**Pull request by alicjakujawa**
## Description

This PR fixes incorrect error which is showing while doing upload. It shows error when form was already submitted and reset. Additionally this PR update list of whitelisted users after removal

**Changes** 🏗

* Updated FileUpload component
* Updated WhitelistedAddresses component

Resolves #3281

### CODE REVIEW

**Comment by alicjakujawa** [comment]
@chinins I was basing from your branch, and made updates to problem i show while testing upload/removal.  We can either merge it to your branch after approval OR i can just wait until you will merge your

**Review by ArmandoGraterol — APPROVED** [review]
I don't see any errors while uploading and reuploading the CSV file, and the whitelist is being updated after removing an user :+1:

**Review by chinins — APPROVED** [review]
Working nicely and correctly. You fixed the error I had in my PR.

As for the merge, I don't think it really matters so whatever you prefer.

**Pull request merged**
